### PR TITLE
Limit the minimum size of the the aggregation grid

### LINKED ISSUE #860: "division by zero" error when using aggregation + attribute columns
I'm trying to instantiate a map that uses the following dataset:

https://cartojs-test.carto.com/tables/ne_10m_populated_places_simple/public

This is the request to Windshaft CartoDB:

```
{
  "buffersize": {
    "mvt": 0
  },
  "layers": [
    {
      "id": "L1",
      "type": "mapnik",
      "options": {
        "cartocss": "\n        #layer {\n          marker-width: 7;\n          marker-fill: ramp([population], (#ecda9a, #f7945d, #ee4d5a), jenks());\n          marker-line-color: #FFFFFF;\n        }\n      ",
        "cartocss_version": "2.1.0",
        "source": {
          "id": "S1"
        },
        "interactivity": [
          "cartodb_id",
          "population"
        ],
        "attributes": {
          "id": "cartodb_id",
          "columns": [
            "population"
          ]
        },
        "minzoom": 0,
        "aggregation": {
          "threshold": 1,
          "resolution": 4,
          "placement": "point-sample",
          "columns": {
            "population": {
              "aggregate_function": "sum",
              "aggregated_column": "pop_max"
            }
          }
        }
      }
    }
  ],
  "dataviews": {},
  "analyses": [
    {
      "id": "S1",
      "type": "source",
      "params": {
        "query": "SELECT * from \n        ne_10m_populated_places_simple\n      "
      }
    }
  ]
}
```

It's basically aggregating by `population` and also specifying `population` in `attributions.columns`.

Windshaft is returning the following error:

```
{
  "errors": [
    "division by zero"
  ],
  "errors_with_context": [
    {
      "type": "layer",
      "message": "division by zero",
      "layer": {
        "id": "L1",
        "index": 0,
        "type": "mapnik"
      }
    }
  ]
}
```

cc: @jgoizueta

### PR DESCRIPTION
**Pull request by jgoizueta**
Fixes #860

### CODE REVIEW

**Comment by jgoizueta** [comment]
Now we should add some test, either to check that `!scale_denominator!` can have a zero value in the presence of aggregations or to check that MapConfig with both attributes and aggregation are instantiated correctly (or both).

File: lib/cartodb/models/aggregation/aggregation-query.js [hunk 1]
```diff
@@ -131,7 +131,10 @@ const dimensionDefs = ctx => {
 // This is equivalent to `${256/ctx.res}*CDB_XYZ_Resolution(CDB_ZoomFromScale(!scale_denominator!))`
 // This is defined by the ctx.res parameter, which is the number of grid cells per tile linear dimension
 // (i.e. each tile is divided into ctx.res*ctx.res cells).
-const gridResolution = ctx => `(${256*0.00028/ctx.res}*!scale_denominator!)::double precision`;
+const gridResolution = ctx => {
+    const minimumResolution = 2*Math.PI*6378137/Math.pow(2,38);
```
[the remark this answers is not in the document]
Comment (jgoizueta) [reply lib/cartodb/models/aggregation/aggregation-query.js]: Do you think the comment I added in the last commit is enough?

[the remark this answers is not in the document]
Comment (enekid) [reply lib/cartodb/models/aggregation/aggregation-query.js]: LGTM!

**Comment by dgaubert** [comment]
👏 

VAMOS!
